include/alpaka/block/sync/Traits.hpp

```diff
@@ -57,62 +57,57 @@ namespace alpaka
             blockSync);
     }
 
-    //-----------------------------------------------------------------------------
-    //! Defines block operation functors.
-    namespace blockOp
+    //#############################################################################
+    //! The addition function object.
```

Comment:
Is it really `addition`? To me seems more like counting.

Reply:
Well, I just took the existing comment. But you are right, counting would be more accurate.